app/models/api/config.rb

```diff
@@ -0,0 +1,17 @@
+class Api::Config
```

Comment:
Sorry I'm pounding you with email but you got my brain churning 😄 

So, another thought that occurred to me is if you use ::Config::Options you may want to name it `Api::Settings`.  This mirrors the toplevel `Settings` class


BTW, I tried locally and it works great:

```ruby
[3] pry(main)> module Api
[3] pry(main)*   Config = ::Config::Options.new.tap {|o| o.add_source!(Rails.root.join("config/api.yml").to_s); o.load!}
[3] pry(main)* end
=> #<Config::Options base=#<Config::Options module="api", name="API", description="REST API", version="2.3.0-pre">, version=#<Config::Options regex=/^v[0-9]+(?>\\.[0-9a-zA-Z]+)*(-[0-9A-Za-z-]+(\\.[0-9A-Za-z-]+)*
[4] pry(main)> Api::Config.collections
=> #<Config::Options accounts=#<Config::Options description="Accounts", options=[:subcollection], methods=[:get], klass="Account">, auth=#<Config::Options description="REST API Authentication", options=[:primary
[6] pry(main)> Api::Config.collections.accounts.klass
=> "Account"
```